policies/equality.md

```diff
@@ -0,0 +1,111 @@
+* Provide equality training and guidance as appropriate, including training on induction and management courses.
+* Ensure that those who are involved in assessing candidates for recruitment or promotion will be trained in non-discriminatory selection techniques.
+* Incorporate equal opportunities notices into general communications practices.
+* Obtain commitments from other persons or organisations such as subcontractors or agencies that they too will comply with the policy in their dealings with our organisation and our workforce.
+* Ensure that adequate resources are made available to fulfil the objectives of the policy.
+
+#  Monitoring and review
+
+We will establish appropriate information and monitoring systems to assist the effective implementation of our equal opportunities policy.
+
+The effectiveness of our equal opportunities policy will be reviewed regularly – at least every 3 years – and action taken as necessary. For example, where monitoring identifies an under-representation of a particular group or groups, we shall develop an action plan to address the imbalance.
+
+#  Complaints
+
+Members who believe that they have suffered any form of discrimination, harassment or victimisation are entitled to raise the matter through the agreed procedures, as stated in Member documentation. A copy of these procedures is available from the Directors. All complaints of discrimination will be dealt with seriously, promptly and confidentially.
+
+In addition to our internal procedures, Members have the right to pursue complaints of discrimination to an industrial tribunal or the Fair Employment Tribunal under the following anti-discrimination legislation:
+
+* [Sex Discrimination (Northern Ireland) Order 1976](https://www.legislation.gov.uk/nisi/1976/1042/contents), as amended
+* [Disability Discrimination Act 1995](https://www.legislation.gov.uk/ukpga/1995/50/contents), as amended
+* [Race Relations (Northern Ireland) Order 1997](https://www.legislation.gov.uk/nisr/2012/263/contents/made), as amended
+* [Employment Equality (Sexual Orientation) Regulations (Northern Ireland) 2003](https://www.legislation.gov.uk/uksi/2004/2519/contents/made)
+* [Fair Employment and Treatment (Northern Ireland) Order 1998](https://www.legislation.gov.uk/nisi/1998/3162/contents), as amended
+* [Employment Equality (Age) Regulations (Northern Ireland) 2006](https://www.legislation.gov.uk/nisr/2006/261/contents/made)
+* [Equal Pay Act (Northern Ireland) 1970](https://www.legislation.gov.uk/apni/1970/32/contents), as amended.
+
+However, Members wishing to make a complaint to a tribunal will normally be required to raise their complaint under our internal grievance procedures first.
+
+Every effort will be made to ensure that Members who make complaints will not be victimised. Any complaint of victimisation will be dealt with seriously, promptly and confidentially. Victimisation will result in disciplinary action and may warrant dismissal.
+
```

Comment:
Whilst an Equal Opportunities policy is important, IMO it would be valuable to refactor this into a full Equality/EDI policy with a focus on member safety and conduct